spec/latest/core/index.html

```diff
@@ -490,8 +490,8 @@
             ****"@type": "SupportedProperty"****,
             ****"property"****: "#property", ####// The property####
             ****"required"****: true, ####// Is the property required in a request to be valid?####
-            ****"readonly"****: false, ####// Can the property's value be modified or is it read-only?####
-            ****"writeonly"****: true ####// Can the property's value be retrieved or is it write-only?####
+            ****"readable"****: false, ####// Can the client read the property's value?####
+            ****"writeable"****: true ####// Can the client write the property's value?####
```

Comment:
What about changing the comments to "Can *a* client *retrieve/change* the property's value?"